src/PlaywrightSharp.Tests/BrowserTypeLaunchTests.cs

```diff
@@ -34,18 +34,6 @@ public async Task ShouldRejectAllPromisesWhenBrowserIsClosed()
 
         }
 
-        [PlaywrightTest("browsertype-launch.spec.ts", "should throw if userDataDir option is passed")]
```

Comment:
We still have this test in upstream. If we can't test it let's leave the test and skip it.

Reply:
Ugh, we should figure out a better way at noting this. Having "empty" tests just be there is annoying :-).

Reply:
We could keep it and then take it away with many others we have in the same situation.

Reply:
I kept it, but I moved it lower - because I added it in a separate commit :D